<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,3">
    <item>
     <widget class="QGraphicsView" name="Graphics"/>
    </item>
    <item>
     <widget class="QWidget" name="InformationWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,6,0,1,6">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QGridLayout" name="PointInputLayout" rowstretch="1,0">
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="PointXLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>X</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="PointYLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Y</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QDoubleSpinBox" name="PointXInput">
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="minimum">
               <double>-2000000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>2000000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QDoubleSpinBox" name="PointYInput">
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="minimum">
               <double>-2000000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>2000000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="PointInputButton">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="TableLayout" stretch="30,1">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QGroupBox" name="PointDeleteGroup">
             <property name="title">
              <string>Удалить точку</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,0,0">
              <item>
               <widget class="QLabel" name="PointDeleteLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Номер точки</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="PointDeleteInput">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>2000000000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="PointDeleteButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Удалить</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Изменить точку</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Номер точки</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="PointEditPositionInput">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>187987987</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Новые координаты</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_2" native="true">
                <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>X</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="PointEdirXInput">
                   <property name="minimum">
                    <double>-2000000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>2000000000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Y</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDoubleSpinBox" name="PointEditYInput">
                   <property name="minimum">
                    <double>-2000000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>2000000000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Изменить</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0" rowspan="3">
            <widget class="QTableWidget" name="PointsTable">
             <property name="columnCount">
              <number>2</number>
             </property>
             <column>
              <property name="text">
               <string>X</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Y</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="PointsDeleteAllButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Удалить все точки</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="SwitchViewButton">
         <property name="text">
          <string>Сменить вид</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="TaskLabel">
         <property name="text">
          <string>Найти такой треугольник с вершинами в точках, у которого разность его площади и площади описанного круга минимальна.  </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="SolutionLayout">
         <item>
          <widget class="QPushButton" name="SolutionButton">
           <property name="text">
            <string>Решение</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="SolutionText"/>
         </item>
         <item>
          <widget class="QPushButton" name="SolutionClearButton">
           <property name="text">
            <string>Очистить вывод</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
